packages/@okta/vuepress-site/docs/guides/implement-oauth-for-okta/main/index.md

```diff
@@ -43,17 +43,18 @@ Create the client application that you want to use with the Okta APIs.
 
 1. In the Admin Console, go to **Applications** > **Applications**.
 1. Click **Create App Integration**.
-1. On the Create a new app integration page, select **OIDC - OpenID Connect** as the **Sign-in method**. Choose any **Application type**. Creating a web, single-page, or native app is an easy way to test scope-based access to Okta's APIs using an OAuth 2.0 bearer token. Click **Next**.
+1. On the **Create a new app integration** page, select **OIDC - OpenID Connect** as the **Sign-in method**. Choose any **Application type**. Creating a web, single-page, or native app is an easy way to test scope-based access to Okta's APIs using an OAuth 2.0 bearer token. Click **Next**.
 
     > **Note:** It is important to choose the appropriate application type for apps that are public clients. Failing to do so may result in Okta API endpoints attempting to verify an app's client secret, which public clients are not designed to have, and would break the sign-in or sign-out flow.
 
-1. Enter a name for your app integration. In the **Sign-in redirect URIs** box, specify the callback location where Okta returns a browser (along with the token) after the user finishes authenticating.
-1. We recommend that you always use the Authorization Code grant flow and clear the **Implicit (Hybrid)** check box.
+1. Enter a name for your app integration. In the **Sign-in redirect URIs** box, specify the callback location where Okta returns a browser (along with the token) after the user finishes authenticating. You can use the default URI for this exercise.
```

Comment:
In the UI, **Grant type** section appears before the **Sign-in redirect URIs** section. Not a major snag, but thinking like a new user and seeing them in a slightly different order threw me a bit.

Reply:
Agree. Will update this.